src/components/Preview/Preview.tsx

```diff
@@ -76,6 +77,9 @@ class Preview extends React.Component<Props & CollectedProps> {
     try {
       isDCLInitialized = true
       window.devicePixelRatio = 1 // without this unity blows up majestically 💥🌈🦄🔥🤷🏼‍♂️
+      await injectScript(`${PUBLIC_URL}/unity/Build/hls.min.js`)
```

Comment:
We can launch all of them at the same time, right? it doesn't need to be in a sequence I think 

```suggestion
      const scriptsURLs = ['unity/Build/hls.min.js', 'editor.js', 'UnityLoader.js']
      await Promise.all<void>(scriptsURLs.map(script => injectScript(`${PUBLIC_URL}/${script}`)))
```

wdyt?

Reply:
It works like a charm!